src/style.css

```diff
@@ -0,0 +1,108 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(.\fonts\Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 700;
+  src: url(.\fonts\Avenir-Heavy.ttf);
+}
+
+html,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.search-form__big {
+  margin: 20px 0;
+}
+
+.search-bar {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.search-bar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search-bar:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-family: Avenir, sans-serif;
+  font-weight: 700;
+}
+
+.search-form__big-input {
+  height: 70px;
+  padding-left: 62px;
+}
+
+.search-form__big-input:focus {
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.search-form__big--label {
+  position: relative;
+}
+
+.search-form__big-input::placeholder {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.search-form__big--label::before {
+  content: "";
+  position: absolute;
+  background-image: url("./images/Search.svg");
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 0;
+}
+
+.search-bar__small {
+  position: relative;
+}
+
+.search-form__small-input {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+  line-height: 19px;
+  background-image: url("./images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: 11px;
+  background-position: 12px 14px;
+}
+
+.search-form__small-input:focus {
+  font-size: 14px;
+  line-height: 19px;
+  color: #3d4e61;
+}
+
+.search-form__small-input::placeholder {
+  font-family: Avenir, sans-serif;
+  font-size: 14px;
+  line-height: 19px;
+  color: #3d4e61;
+}
```

Comment:
Don't repeat yourself. If the same styles are used on different elements, then you need to bring these styles into a common class and use this class in both elements.